src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/Import.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace ILCompiler.DependencyAnalysis.ReadyToRun
+{
+    public abstract class Import : EmbeddedObjectNode
+    {
+        public abstract EmbeddedPointerIndirectionNode<Signature> GetSignature(NodeFactory factory);
```

Comment:
Would it be more flexible if this returned `Signature` instead of `EmbeddedPointerIndirectionNode<Signature>` and the caller wrapped this into whatever kind of indirection it needs (should it need one)?

Reply:
I altered it to return Signature - looks better